### DISCUSSION TITLE
[FEATURE REQUEST] Adjust crowd control of comments

### DISCUSSION THREAD

**Discussion by atomicpoet** [mastodon/mastodon#21451] [General]
Create the ability to adjust crowd control after a thread is created. This would allow you to hide comments from accounts you're not sure about, protecting the community from malicious actors. Crowd control could have the following levels:

* Off
* Lenient
* Moderate
* Strict

**Comment by jaanus** [comment]
I don’t understand what this means. On what object is this setting? (I assume the original post/thread) and what does it actually do?
